virt-v2v/cold/entrypoint.go

```diff
@@ -73,22 +119,13 @@ func main() {
 	}
 
 	if source == vSphere {
-		if _, err := os.Stat("/etc/secret/cacert"); err == nil {
-			// use the specified certificate
-			err = os.Symlink("/etc/secret/cacert", "/opt/ca-bundle.crt")
-			if err != nil {
-				fmt.Println("Error creating ca cert link ", err)
-				os.Exit(1)
-			}
-		} else {
-			// otherwise, keep system pool certificates
-			err := os.Symlink("/etc/pki/tls/certs/ca-bundle.crt.bak", "/opt/ca-bundle.crt")
-			if err != nil {
-				fmt.Println("Error creating ca cert link ", err)
-				os.Exit(1)
+		virtV2vArgs = append(virtV2vArgs, "-ip", "/etc/secret/secretKey")
+
+		if checkEnvVariablesSet("V2V_staticIPs") {
```

Comment:
really, what's the point in calling os.Getenv twice? you need the value of V2V_staticIPs here, so why not getting it and then check it to avoid another system call?..